apps/sms/views/shared/style/panel.css

```diff
@@ -5,34 +5,51 @@
   top: 0;
   bottom: 0;
 
-  transform: translate(100%);
-
   overflow: auto;
 
   background: white;
 
   animation: .2s both linear;
 }
+html[dir="ltr"] .panel { transform: translate(100%); }
+html[dir="rtl"] .panel { transform: translate(-100%); }
 
-.panel-active {
+/**
+ * adding html[dir] to make the specificity of this rule high enough to
+ * overwrite the default translation for element with class panel, without
+ * needing to put a !important
+ */
+html[dir] .panel-active {
   transform: translate(0);
 }
 
-.panel-will-activate {
+html[dir="ltr"] .panel-will-activate {
   animation-name: new-slide-left;
 }
+html[dir="rtl"] .panel-will-activate {
+  animation-name: new-slide-right;
+}
 
-.panel-will-activate.panel-animation-reverse {
+html[dir="ltr"] .panel-will-activate.panel-animation-reverse {
   animation-name: new-slide-right;
 }
+html[dir="rtl"] .panel-will-activate.panel-animation-reverse {
+  animation-name: new-slide-left;
+}
 
-.panel-will-deactivate {
+html[dir="ltr"] .panel-will-deactivate {
   animation-name: old-slide-left;
 }
+html[dir="rtl"] .panel-will-deactivate {
+  animation-name: old-slide-right
+}
 
-.panel-will-deactivate.panel-animation-reverse {
+html[dir="ltr"] .panel-will-deactivate.panel-animation-reverse {
   animation-name: old-slide-right;
 }
+html[dir="rtl"] .panel-will-deactivate.panel-animation-reverse {
+  animation-name: old-slide-left;
+}
```

Comment:
Mmm I don't think these classes are used at all, actually. I think this comes from one of the earlier versions of bug 1162030... I changed using `style.animationName` because it was faster in one of the latest iterations of the patch, and that's why these css rules are still there.

Maybe you could simply remove all this ?